src/main/java/uk/gov/hmcts/reform/professionalapi/configuration/resolver/OrganisationIdArgumentResolver.java

```diff
@@ -56,7 +60,7 @@ public Object resolveArgument(
         ProfessionalUser professionalUser;
         Organisation organisation;
 
-        UserInfo userInfo = jwtGrantedAuthoritiesConverter.getUserInfo();
+        UserInfo userInfo = idamRepository.getUserInfo(getUserToken());

```

Comment:
The code suggests that loading the userInfo from IDAM is required only to get the unique ID of the user - this information is already present in the userToken (JWT). A simpler solution is to parse the JWT and get the `uid` claim from there.